.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -2,6 +2,8 @@
 - [ ] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/brew/pulls) for the same change?
 - [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
 - [ ] Have you written new tests for your changes? [Here's an example](https://github.com/Homebrew/homebrew/pull/49031).
+- [ ] Have you successfully run `brew style` with your changes locally?
+- [ ] Have you successfully run `brew doctor` with your changes locally?
```

Comment:
there are too many benign things flagged by doctor so let's not add that bullet

Reply:
ok, removing that row again. While CI seems to fail if `brew doctor` fails, it's maybe unusual for doctor to fail due to new changes.